### DISCUSSION TITLE
import existing blog feed

### DISCUSSION THREAD

**Discussion by phsung** [microfeed/microfeed#94] [General] [answered]
We have an existing blog (for podcast) and would like to try microfeed. Is there anyway to import an existing rss feed into microfeed? Thanks!

-Patrick

**Comment by wenbinf** [comment, marked as answer]
Each instance of microfeed comes with an API, accessible at [Microfeed's API documentation](https://www.microfeed.org/json/openapi#tag/Admin-API/operation/apiCreateNewItem).

Should you wish to migrate content to Microfeed, you can parse your existing RSS feed and utilize this API to automate the process.

For blogs with only a few posts, it may be faster to manually copy and paste the content rather than writing a script to handle the migration.

(Note: We recognize the value of providing a dedicated tool or script to simplify the migration process and will consider implementing this in the future.)

**Reply by amblerkr** [reply]
I'm a non-developer and it would be great to be able to automate this using automation services like IFTTT and Zapier.

**Comment by wenbinf** [comment]
> I'm a non-developer and it would be great to be able to automate this using automation services like IFTTT and Zapier.

I think it's doable via Zapier or the like, as long as those tools can read & parse REST Api's json response. 

After some quick google search, here's what I found - https://zapier.com/blog/how-to-use-api/

**Reply by amblerkr** [reply]
I'll study it. I'll try this, thank you so much.
